cmd/multitenant-autoscaler/main.go

```diff
@@ -28,9 +28,9 @@ import (
 	"github.com/knative/serving/pkg/autoscaler/statserver"
 	clientset "github.com/knative/serving/pkg/client/clientset/versioned"
 	informers "github.com/knative/serving/pkg/client/informers/externalversions"
-	reconciler "github.com/knative/serving/pkg/controller"
-	"github.com/knative/serving/pkg/controller/autoscaling"
 	"github.com/knative/serving/pkg/logging"
+	reconciler "github.com/knative/serving/pkg/reconciler"
```

Comment:
This import identifier is redundant now. (here and in most files)

Reply:
Oops, cleanup up now.